The codes building issue of unexpected keyword argument 'follow_symlinks'

pull the latest codes from TheRock, and run the below commands to build codes
1)  pip install CppHeaderParser==2.7.4 meson==1.7.0 
2) sudo apt install gfortran git-lfs ninja-build cmake g++ pkg-config xxd libgtest-dev
3) install cmake 3.26.0
4) install patchelf-0.18.0 
5) cmake -B build -GNinja . -DTHEROCK_AMDGPU_FAMILIES=gfx110X-dgpu 
6) cmake --build build
 
but met the below issue:
[therock-numactl] ++ Exec [/home/TheRock/build/third-party/sysdeps/linux/numactl/build/build]$ /usr/local/bin/patchelf --set-soname librocm_sysdeps_numa.so.1 /home/TheRock/build/third-party/sysdeps/linux/numactl/build/stage/lib/rocm_sysdeps/lib/librocm_sysdeps_numa.so.1
[therock-numactl] Traceback (most recent call last):
[therock-numactl]   File "/home/TheRock/build_tools/patch_linux_so.py", line 113, in <module>
[therock-numactl]     main(sys.argv[1:])
[therock-numactl]   File "/home/TheRock/build_tools/patch_linux_so.py", line 109, in main
[therock-numactl]     run(args)
[therock-numactl]   File "/home/TheRock/build_tools/patch_linux_so.py", line 94, in run
[therock-numactl]     add_prefix(args)
[therock-numactl]   File "/home/TheRock/build_tools/patch_linux_so.py", line 70, in add_prefix
[therock-numactl]     if orig_path.exists(follow_symlinks=False):
[therock-numactl] TypeError: Path.exists() got an unexpected keyword argument 'follow_symlinks'
[therock-numactl] ninja: build stopped: subcommand failed.
ninja: build stopped: subcommand failed.
 
Code commit ID:commit a41fef22eda3b0ffdca0378ee0ee205d3791b4f0 
OS: ubuntu 22.04
Python version: python3.10 
GPU: AMD Radeon w7900

--- a/build_tools/patch_linux_so.py
+++ b/build_tools/patch_linux_so.py
@@ -67,7 +67,7 @@ def add_prefix(args: argparse.Namespace):
 
         # Remove old links.
         for orig_path in orig_paths:
-            if orig_path.exists(follow_symlinks=False):
+            if orig_path.is_symlink():
                 print(f"Removing original link: {orig_path}")
                 orig_path.unlink()
 

